### ISSUE TITLE
bug: Monet theming has contrast issues

### ISSUE THREAD

**Issue by salarua** [GradienceTeam/Gradience#425]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### What happened?

The destructive and warning styles, in addition to the accent style in light mode, have contrast issues.

### To Reproduce

1. Apply Monet styling
2. Open Adwaita Demo
3. Open Style Classes demo to observe contrast issues

### Screenshots

Input image: 

![timothy-eberly-HUiNRjXr-bQ-unsplash](https://user-images.githubusercontent.com/104225819/190282260-fe542126-f1b7-421a-ad7b-fc814197c8fc.jpg)

Light mode: 

![Screenshot from 2022-09-14 16-37-35](https://user-images.githubusercontent.com/104225819/190282194-d4b0bd18-6d4f-44fb-b9d2-51bbbf710610.png)

Dark mode: 

![Screenshot from 2022-09-14 16-38-29](https://user-images.githubusercontent.com/104225819/190282220-a497a418-4fe4-4a1c-8fd5-5a1c27e80161.png)

---

Input image:

![DESKTOP-5K-FREE-21 04 22165839-by-@WALLSBYJFL](https://user-images.githubusercontent.com/104225819/190282345-7be1491b-c3d8-4878-9b12-654846dccf7c.jpg)

Light mode: 

![Screenshot from 2022-09-14 16-40-16](https://user-images.githubusercontent.com/104225819/190282389-e22d0388-574d-4405-842b-3af9cffa1f7f.png)

Dark mode: 

![Screenshot from 2022-09-14 16-40-59](https://user-images.githubusercontent.com/104225819/190282405-1db139cd-b945-489a-acec-17d89c3c7380.png)

---

Input image:

![ng-32703](https://user-images.githubusercontent.com/104225819/190282537-bb7ca6e0-b6f4-475d-9a54-74b5c9c32c04.jpg)

Light mode:

![Screenshot from 2022-09-14 16-42-35](https://user-images.githubusercontent.com/104225819/190282575-adcc9ee2-f36d-4a21-804d-f09088130ff2.png)

Dark mode: 
![Screenshot from 2022-09-14 16-43-41](https://user-images.githubusercontent.com/104225819/190282584-2fe29a52-0aa1-4c11-a65c-3afe5638dd63.png)

Notice how the warning style is not distinct enough from the background, and how the text on the suggested and destructive buttons are not easily visible against the background color.

### OS

Pop!_OS 22.04 LTS

### Version

0.2.2

### Installation method

Flatpak from Flathub

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by AdwCustomizer** [comment]
Welcome on Gradience. 🥳 We really appreciate your contribution. The core team will review your issue as soon as possible. You can also join the matrix room https://matrix.to/#/#Gradience:matrix.org or the discord server https://discord.com/invite/rwNDGPJf

**Comment by 0xMRTT** [comment]
Thanks for reporting this issue. I'll fix that soon. 

Did you tried to change the tone ? Maybe it'll slove your issue

**Comment by salarua** [comment]
No. Whenever I change the tone, the Monet palette does change, but the colors actually used in the theme don't. Should I open a separate issue for this?

**Comment by 0xMRTT** [comment]
Currently, the palette displayed at the bottom is just a preview of the generated theme

**Comment by salarua** [comment]
Yes, but it doesn't actually change anything. If I apply tone 50, the theme looks the same as tone 20.

**Comment by salarua** [comment]
This bug seems to have reappeared, as destructive buttons no longer have sufficient contrast in light or dark mode.

**Comment by salarua** [comment]
I can confirm commit ac4e68f reintroduced this bug.
